test/e2e/flowaggregator_test.go

```diff
@@ -609,30 +615,26 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 				assert.Contains(record, fmt.Sprintf("egressNetworkPolicyRuleAction: %d", ipfixregistry.NetworkPolicyRuleActionAllow), "Record does not have the correct NetworkPolicy RuleAction with the egress rule")
 			}
 
-			// Skip the bandwidth check for the iperf control flow records which have 0 delta count.
-			if checkBandwidth && !strings.Contains(record, "octetDeltaCount: 0") {
-				exportTime := int64(getUnit64FieldFromRecord(t, record, "flowEndSeconds"))
-				curOctetTotalCount := getUnit64FieldFromRecord(t, record, "octetTotalCountFromSourceNode")
+			// Skip the bandwidth check for the iperf control flow records which have 0 throughput.
+			if checkBandwidth && !strings.Contains(record, "throughput: 0") {
 				flowStartTime := int64(getUnit64FieldFromRecord(t, record, "flowStartSeconds"))
-				if curOctetTotalCount > octetTotalCount {
-					octetTotalCount = curOctetTotalCount
-				}
-				curOctetDeltaCount := getUnit64FieldFromRecord(t, record, "octetDeltaCountFromSourceNode")
-				// Check the bandwidth using octetDeltaCountFromSourceNode, if this record
-				// is neither the first record nor the last in the stream of records.
-				if curOctetDeltaCount != curOctetTotalCount && exportTime < flowStartTime+iperfTimeSec {
-					t.Logf("Check the bandwidth using octetDeltaCountFromSourceNode %d in data record.", curOctetDeltaCount)
-					// This middle record should aggregate two records from Flow Exporter
-					checkBandwidthByInterval(t, bandwidthInMbps, curOctetDeltaCount, float64(2*exporterActiveFlowExportTimeout/time.Second), "octetDeltaCountFromSourceNode")
+				exportTime := int64(getUnit64FieldFromRecord(t, record, "flowEndSeconds"))
```

Comment:
not introduced by this PR but there is a typo here: s/getUnit64FieldFromRecord/getUint64FieldFromRecord